Arrays.js

```diff
@@ -0,0 +1,38 @@
+//Дан массив состоящий из названий фильмов, выполните перебор массива с выводом в консоль названия каждого фильма
+let arr = ["Terminator", "Terminator 2", "Terminator 3"];
+    for(let movie of arr) {
+    console.log(movie);
+    };
+
+//Дан массив производителей автомобилей, преобразовать массив в строку и обратно в массив
+let carEng = ["VW", "Dodge", "KIA"];
+let str = carEng.join(", ");
+console.log(str);
+let againArr = str.split(", ");
+console.log(againArr);
+
+//Дан массив имен ваших знакомых, добавить к каждому элементу массива слова hello
+let names = ["Vasya", "Vika", "Inokentiy"];
+let name = names.map((item, index) => (index = true)? (item = "Hello " + item): NaN );
```

Comment:
1) `const`.
2) Can be simplified into `const name = names.map(item => ("Hello " + item));`
3) Use string interpolation.